src/jetson/models/utils/transform.py

```diff
@@ -0,0 +1,31 @@
+import cv2
+import numpy as np
+
+
+class BaseTransform:
+    def __init__(self, size, mean):
+        '''
+        Transform image to desired size before passing to face detector
+
+        Params:
+        size (Type: int) - Desired input size to face detector
+        mean (Type: tuple of length 3 for 3 channels) - mean intensity values to perform normalization.
+
+        '''
```

Comment:
please provide type hinting in the function definition. also replace `size (Type: int)` with `(int)` "Type" and `mean (Type: tuple of ...)` with `mean (tuple)` and move the "length of 3 for 3 channels" into the parameter description. saying `Type: <type>` is redundant language here.